src/Giraffe/ResponseWriters.fs

```diff
@@ -9,16 +10,47 @@ open Microsoft.Net.Http.Headers
 open FSharp.Control.Tasks.V2.ContextInsensitive
 open Giraffe.GiraffeViewEngine
 
+
 // ---------------------------
 // Helper functions
 // ---------------------------
 
+let private DefaultCapacity = 1386
+let private MaxBuilderSize  = DefaultCapacity * 3
+
+type StringBuilderCache =
+
+    [<ThreadStatic>]
+    [<DefaultValue>]
+    static val mutable private instance : StringBuilder
+
+    static member Get () = StringBuilderCache.Get(DefaultCapacity)
+    static member Get (capacity : int) =
```

Comment:
If `nodeToUtf8HtmlDoc` always calls `Get()` do we even need the overload with `capacity : int` ?